lib/src/atoms/file/link.rs

```diff
@@ -55,8 +55,11 @@ impl Atom for Link {
             }
         };
 
+        const PREFIX: &str = r#"\\?\"#;
```

Comment:
I thought about having a construct like:

```rust
        let source= if cfg!(target_os = "windows") {
            const PREFIX: &str = r#"\\?\"#;

            PathBuf::from(&self.source.display().to_string().replace(PREFIX, ""))
        } else {
            self.source.to_owned()
        };

        // If this file doesn't link to what we expect, lets make it so
        !link.eq(&source)
```

to make it narrower depending on the OS.

And maybe we could have a test for this edge case in particular.

What do you think about it?

Reply:
I just tested it without problems. Let's narrow it down.